### PR TITLE
Remove perfCounter. 

### PR DESCRIPTION
**Pull request by EleanorRigby**
Add cluster resource usage summary.

### CODE REVIEW

File: src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb [hunk 1]
```diff
@@ -81,40 +81,22 @@ def select_datastores(pattern)
...
-            percent_used = Util.average_csv(counter['mem.usage.average']) / 10000
-            free_memory = ((1.0 - percent_used) * total_memory).to_i
-          else
-            logger.warn("host '#{host_properties['name']}' is missing 'mem.usage.average' (possibly recently booted), ignoring")
-          end
-
-          cluster_free_memory += free_memory
+      # @return
+      def fetch_cluster_utilization()
+        require 'pry-byebug'
```
Comment (EleanorRigby) [inline src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb]: Have to remove this. Sorry

File: src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb [hunk 2]
```diff
@@ -81,40 +81,22 @@ def select_datastores(pattern)
...
-          if !counter['mem.usage.average'].nil?
-            percent_used = Util.average_csv(counter['mem.usage.average']) / 10000
-            free_memory = ((1.0 - percent_used) * total_memory).to_i
-          else
-            logger.warn("host '#{host_properties['name']}' is missing 'mem.usage.average' (possibly recently booted), ignoring")
-          end
-
-          cluster_free_memory += free_memory
+      # @return
+      def fetch_cluster_utilization()
```
Comment (nehagjain15) [inline src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb]: Method name should be effective_memory as thats what this is returning

**Comment by cppforlife** [comment]
when does `overall_status` change? when is it green, red, etc?

File: src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb (hunk 2 above)
Comment (EleanorRigby) [reply src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb]: This was earlier also named as cluster utilization. I would prefer to keep changes as minimal as possible. If we ever introduce cluster CPU usage as one of the requirements , then I can rename things differently.
What do you think?

**Comment by EleanorRigby** [comment]
@cppforlife : Thanks for having a look at my PR
Overall Status represents a ManagedEntityStatus . 

gray | The status is unknown- Initially the alarms start with the Gray status.
green | The entity is OK and the cluster is healthy.
yellow | This is just a warning but potentially could lead to an alert (RED), but the cluster is still functional. 
red | The cluster has a serious problem and this sets of an alert - VSAN issues.

**Pull request closed without merging**